scripts/lldb_pretty_printers/data_path_pretty_printer.py

```diff
@@ -0,0 +1,36 @@
+import lldb
+
+def data_path_summary(valobj, internal_dict):
+    """
+    Return a string summary for nx::core::DataPath.
+    We'll display the contents of m_Path (std::vector<std::string>)
+    as a list of unquoted string values.
+    """
+    path_valobj = valobj.GetChildMemberWithName('m_Path')
+
+    # Use LLDB's synthetic children for std::vector to retrieve each element.
+    count = path_valobj.GetNumChildren()
+    elements = []
+    for i in range(count):
```

Comment:
I looked at the documentation and fortunately `lldb.SBValue` has `__iter__` so you can do `for element in path_valobj`.